<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Umut Fitness</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_kullanici_adi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hoş geldiniz, kullanici_adi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vki">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hoş geldiniz, vki</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_boy">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Boy (cm) :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_boy">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_kilo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilo (kg) :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_kilo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gonder">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Gönder</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
